pkg/apis/pipeline/v1/param_types.go

```diff
@@ -403,7 +396,7 @@ var AllParamTypes = []ParamType{ParamTypeString, ParamTypeArray, ParamTypeObject
 
 // ParamValues is modeled after IntOrString in kubernetes/apimachinery:
 
-// ParamValue is a type that can hold a single string, string array, or string map.
+// ParamValue is a type that can hold a single string or string array.
```

Comment:
This doesn't look right-- a param value can be a map. v1beta1 is the package that needs to be updated here.